presto-hive/src/main/java/io/prestosql/plugin/hive/parquet/ParquetPageSource.java

```diff
@@ -146,28 +130,12 @@ public Page getNextPage()
                 return null;
             }
 
-            Block[] blocks = new Block[hiveColumnIndexes.length];
+            Block[] blocks = new Block[fields.size()];
             for (int fieldId = 0; fieldId < blocks.length; fieldId++) {
-                if (constantBlocks[fieldId] != null) {
-                    blocks[fieldId] = constantBlocks[fieldId].getRegion(0, batchSize);
-                }
-                else {
-                    Type type = types.get(fieldId);
-                    Optional<Field> field = fields.get(fieldId);
-                    int fieldIndex;
-                    if (useParquetColumnNames) {
-                        fieldIndex = getFieldIndex(fileSchema, columnNames.get(fieldId));
```

Comment:
This method is now unused and can be removed.

Reply:
Do you mean `getFieldIndex`? I've removed it in the commit.

Reply:
I mean the method can be deleted from `ParquetTypeUtils`. Perhaps you already removed it locally but didn't push to GitHub?

Reply:
Oh I see, I didn't remove it from `ParquetTypeUtils`. I thought you meant the call of this method. Will do.